File: program (99).cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    Display                                        
//  Description :       Accept number from user and write a Recursive program and Display numbers and " * " in decreasing order. 
//  Input :             int 
//  Output :            int and char
//  Author :            Aman Shaikh    
//  Date:               15/12/2024 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Pattern
{
    public:
    int Display(int iNo) 
    {
        if(iNo >= 1) 
        {
            printf("%d\t*\t",iNo); 
            iNo--;   
            Display(iNo);              
        } 
    }
};

int main()
{
    int iValue = 0;

    cout<<"Enter the number : "; 
    cin>>iValue;

    cout<<"Output is :\t";

    Pattern pobj;

    pobj.Display(iValue);

    return 0;
}

File: program (99).java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    CountCapital
//  Description :       Count Capital Letters in the hardcoded String. 
//  Input :             String 
//  Output :            String
//  Author :            Aman Shaikh    
//  Date:               14/12/2024 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class StringDemo
{
    public int CountCapital(String str)
    {
        int iCnt = 0, i = 0;
        while(i < str.length()) 
        {
            char ch = str.charAt(i);
            if(ch >= 'A' && ch <= 'Z')
            {
                iCnt++;
            } 
            i++;
        }
        return iCnt;
    }
}

class Assignment36A
{
    public static void main(String Arr[])
    {
        String s1 = "Marvellous Multi OS";
        StringDemo sobj = new StringDemo();
        int iResult = sobj.CountCapital(s1);
        System.out.println("Capital Numbers are : "+iResult);
    }
}